rust/otap-dataflow/crates/admin/src/pipeline.rs

```diff
@@ -39,3 +52,44 @@ pub async fn show_status(
     let pipeline_status = state.observed_state_store.pipeline_status(&key);
     Ok(Json(pipeline_status))
 }
+
+/// Used by the kubelet livenessProbe to decide whether to restart the container.
```

Comment:
Is mentioning that this is specifically for kubelet might be too implementation specific as a description of the endpoints? E.g. other services could make use of these as well. If someone were to run this on ECS, or behind an LB that does active load checking for example.

Also, how is this intended to be used by kube? Will the `pipeline_group_id` and `pipeline_id` be known a-priori? and isn't there multiple of them per instance of the application? kube only allows us to define one liveness probe as far as I know. I guess we'd configure it like this?
```yaml
spec:
  containers
  - name: otap-dataflow
    // ..
    livnessProbe:
      httpGet:
        path: /pipeline-groups/known_pipeline_group_id/pipelines/known_pipeline_id/livez
        port: liveness-port
```